flocker/control/_persistence.py

```diff
@@ -0,0 +1,92 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Persistence of cluster configuration.
+"""
+
+from pickle import dumps, loads
+
+from twisted.application.service import Service
+from twisted.internet.defer import succeed
+
+from ._model import Deployment
+
+
+# These should not use Pickle!@!
+# https://clusterhq.atlassian.net/browse/FLOC-1241x
+def serialize_deployment(deployment):
+    """
+    Convert a ``Deployment`` object to ``bytes``.
+
+    :param Deployment deployment: Object to serialize.
+
+    :return bytes: Serialized object.
+    """
+    return dumps(deployment)
+
+
+def deserialize_deployment(data):
```

Comment:
Hm. I feel like we should be doing FLOC-1214 ASAP (we'll be using pickle over the network otherwise :fearful:), which will solve that. Only reason I didn't do it first is that it's easily parallelizable with the rest of the work, so didn't seem worth making it a bottleneck.